### PR TITLE
Select2 into core

### PR DESCRIPTION
**Pull request by skjnldsv**
@jancborchardt

You took too much time, Shia is disappointed! 
bkanf7d

### CODE REVIEW

File: lib/private/legacy/template.php [hunk 1]
```diff
@@ -117,7 +117,7 @@ public static function initTemplateEngine($renderAs) {
 			OC_Util::addStyle("fonts",null,true);
 			OC_Util::addStyle("icons",null,true);
 			OC_Util::addStyle("header",null,true);
-			OC_Util::addStyle("inputs",null,true);
+			OC_Util::addStyle("inputs");
```
Comment (nickvergessen) [inline lib/private/legacy/template.php]: unrelated?

Comment (skjnldsv) [inline lib/private/legacy/template.php]: No no related. See https://github.com/nextcloud/server/pull/1789#issuecomment-254592977

**Comment by MorrisJobke** [comment]
@skjnldsv Could I ask you to squash all your commits into one? 😉

**Comment by skjnldsv** [comment]
You could! :joy:

**Comment by MorrisJobke** [comment]
> You could! 😂

Will you do so? 😝

**Comment by skjnldsv** [comment]
Your desire are orders! 😆

**Comment by MorrisJobke** [comment]
```
select2-toggleselect.js?v=42c21b7…:53 Uncaught ReferenceError: Select2 is not defined(anonymous function) @ select2-toggleselect.js?v=42c21b7…:53
```

**Comment by MorrisJobke** [comment]
I got this error in the JS console when opening the files app.

**Comment by MorrisJobke** [comment]
Sorry I need to remove the emoji - crashed drone

**Comment by skjnldsv** [comment]
😭 
This is strange, I don't have the error.

**Comment by MorrisJobke** [comment]
> This is strange, I don't have the error.

You only see it on the error log 😉

**Comment by skjnldsv** [comment]
Found it, I didn't had the same file on my test server. Fixed!

**Comment by MorrisJobke** [comment]
> Found it, I didn't had the same file on my test server. Fixed!

It's not about crashing the actual build, but drone and especially the mysql behind the scenes failed to store the emoji, because mysql can only store 3 byte characters and emojis are 4 byte characters 🙈

**Comment by skjnldsv** [comment]
Ahahaa, this is a funny issue! :p 

Anyhow, review pls 😉

**Comment by rullzer** [comment]
@skjnldsv you need to sign off your commit ;)

**Comment by jancborchardt** [comment]
@skjnldsv you can sign off your commits by adding -s when doing the commit. So `git commit -sm "comment"`

You can fix this on an already committed thing by using `git commit --amend -s` and then force pushing to this branch.

**Comment by rullzer** [comment]
@skjnldsv it has a different purpose. See https://github.com/nextcloud/server/blob/master/contribute/developer-certificate-of-origin

**Comment by jancborchardt** [comment]
@skjnldsv yeah, sorry it’s a small addition in the workflow. :) At least you don’t need to sign a CLA and transfer all the rights to a company. :wink:

**Comment by skjnldsv** [comment]
No need to apologise! :D

**Comment by rullzer** [comment]
Works!
LGTM

**Comment by LukasReschke** [comment]
LGTM
